Modules/Microsoft365DSC/DSCResources/MSFT_TeamsGuestMeetingConfiguration/MSFT_TeamsGuestMeetingConfiguration.psm1

```diff
@@ -69,8 +72,7 @@ function Get-TargetResource
         Identity = 'Global'
     }
 
-    try
-    {
+    try {
```

Comment:
Please make sure you don't update the code styling formatting. This is done in several instances 😉

Reply:
In the same code file, I see the brackets used in a mixed way. Sometimes
try {
}
and sometimes
try
{
}
I think a single file should at least follow the same format, shouldn't it? My VS Code automatically adjust these. 🙂

Reply:
That indeed should be consistent across all files. Within M365DSC we are using the "open braces on the next line" format, we have VSCode settings included in the project. Your VS Code should automatically use that:
https://github.com/microsoft/Microsoft365DSC/blob/93937c70b5882ddfd6ce6cf16ba701ccd3e9761b/.vscode/settings.json#L22

It is possible that the wrong format is used. Most probably because we missed that in a past reviews 😉

Reply:
I will reformat then all the files this way. 😉